apps/museums/fields/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   },
   'contact-phone': {
     mixin: 'input-text',
-    validate: ['required', 'phonenumber']
+    validate: 'required'
```

Comment:
How would you want to present the phone number back to the user (assuming you will need to do so on the confirm page)? If you need/would like to present it as it was input, formatting would need to occur twice. I vote for changing the formatter